README.md

```diff
@@ -3,41 +3,59 @@ Repository faking an add-ons server for [@widelands](https://github.com/wideland
 
 I don’t know how long this "server" will be in use, so I’d better write some brief documentation about maintenance here.
 
+[Informations how to create add-ons can be found here.](https://github.com/widelands/widelands/blob/master/doc/sphinx/source/add-ons.rst)
+
+If you do not wish to create a pull request for a new add-on here, it is also possible to submit add-ons on the forum: https://www.widelands.org/forum/topic/5073
+
+### Prerequisites
+
+Before adding add-ons to this repository be sure a java development kit (JDK) is installed to compile the UpdateList.java script.
+
+After the installation of the JDK run `javac UpdateList.java` in the base folder.
+
 ### Adding a new add-on
 
-Just add it to the `addons/` folder, then run `java UpdateList`, then git add,commit,push.
+#### Addon
 
-### Updating an add-on
+Just add the folder `name_of_addon.wad` to the `addons/` folder.
 
-Just replace the add-on’s entry in the `addons/` folder with the new one, then run `java UpdateList`, then git add,commit,push.
+#### Screenshots
 
-### Updating screenshots
+If you want to provide some screenshots for your addon, add the screenshots to the subfolder `screenshots/name_of_addon.wad` and create a file called `descriptions`. Refer to the examples to get used to this file.
 
-When you updated any of the files in `screenshots/` or added new ones, just run `java UpdateList`, then git add,commit,push. Be sure to update the `descriptions` file beforehand.
+#### Apply changes
 
-### Updating translations
+After adding the addon and probably some screenshots, run `java UpdateList` inside the base folder, then git add,commit,push. Then create a pull request here.
 
-When you updated any of the files in `po/` or added new ones, just do git add,commit,push.
+### Updating an add-on
 
-A GitHub action automatically compiles the translations periodically by running `buildcats.sh`. **Do not** run this script manually. If you have to do it anyway for some reason, do this **only** on the master branch.
+Just replace the folder below `addons` with the new one.
+Same applies to screenshots: Just replace or add new ones. Be sure to update the `descriptions` file.
+
+After you have done run `java UpdateList`, then git add,commit,push.
 
 ### Verifying add-ons
 
-Run `java UpdateList '/cool_feature.wad'` (don’t forget the '/' before the add-on’s name!), then run `java UpdateList`, then git add,commit,push.
+Updating an add-on will automatically set it to Not Verified. It is not allowed to mark an add-on as verified during the update.
+To verify it run `java UpdateList '/cool_feature.wad'` (don’t forget the '/' before the add-on’s name!), then run `java UpdateList`, then git add,commit,push.
+
```

Comment:
I verifying done automatically?

Reply:
Verifying means that a developer looks at the add-on in question and decides that it is safe for use; then he runs this script and makes a commit with the change. The add-on will then get the Verified badge in the add-ons manager.